var/spack/repos/builtin/packages/xbraid/package.py

```diff
@@ -14,15 +14,49 @@ class Xbraid(MakefilePackage):
     homepage = "https://computing.llnl.gov/projects/parallel-time-integration-multigrid/software"
     url      = "https://github.com/XBraid/xbraid/archive/v2.2.0.tar.gz"
 
+    version('3.0.0', sha256='06988c0599cd100d3b3f3ebb183c9ad34a4021922e0896815cbedc659aaadce6')
+    version('2.3.0', sha256='706f0acde201c7c336ade3604679759752a74e2cd6c2a29a8bf5676b6e54b704')
     version('2.2.0', sha256='082623b2ddcd2150b3ace65b96c1e00be637876ec6c94dc8fefda88743b35ba3')
 
     depends_on('mpi')
 
+    @when('@:2.2.0')
     def build(self, spec, prefix):
         make('libbraid.a')
 
+    @when('@2.3.0:')
+    def build(self, spec, prefix):
+        make('braid')
+
     # XBraid doesn't have a real install target, so it has to be done
     # manually
+    @when('@2.3.0:')
+    def install(self, spec, prefix):
+        # Install headers
+        mkdirp(prefix.include)
+        headers = glob.glob('braid/*.h')
+        headers.extend(glob.glob('braid/*.hpp'))
+        for f in headers:
+            install(f, join_path(prefix.include, os.path.basename(f)))
```

Comment:
`install` and `install_tree` now support glob characters, so it may be possible to simplify this. Does this work?
```python
install('braid/*.h', prefix.include)
install('braid/*.hpp', prefix.include)
```
If so you can update the older code too.

Reply:
This works, so I have updated both sections to use this.